lessons/advanced/gen-stage-1.md

```diff
@@ -0,0 +1,368 @@
+---
```

Comment:
ie the next one that I just finished the code for in the linked example repo in the latest commits could be `gen-stage-2` then `gen-stage-3` if it went that far etc.